extensions/stacks/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "stacks",
+  "title": "Stacks",
+  "description": "Transform your digital footprint into personal intelligence",
+  "icon": "extension-icon.png",
+  "author": "sourabh_rathour",
+  "categories": [
+    "Applications",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-resources",
+      "title": "Stacks Search",
+      "subtitle": "Grid view of your resources",
+      "description": "Search your saved resources in a grid view",
+      "mode": "view"
+    },
+    {
+      "name": "search-resources-list",
+      "title": "Stacks Search List",
+      "subtitle": "Detailed list view of your resources",
+      "description": "Search your saved resources in a detailed list view with previews",
+      "mode": "view"
+    }
```

Comment:
Could this be one command instead with a preference value to select `Grid` or `List`